Framework/include/QualityControl/runnerUtils.h

```diff
@@ -119,6 +119,43 @@ inline std::string computeProvenance(const std::string& fallbackProvenance = "")
   return provenance;
 }
 
+inline std::string indentTree(int level)
+{
+  std::string s;
+  for (int i = 0; i < level; i++)
+    s += "  ";
+  return s;
+}
+
+inline void printTree(boost::property_tree::ptree& pt, int level = 0)
```

Comment:
I can't quite follow the whole logic, but does it work for arrays? I don't see any `[` and `]` characters.

Reply:
Yes, I am not sure why I went this route. I have replaced it with the standard `json_parser`